src/providers/index.ts

```diff
@@ -42,6 +42,7 @@ import HuggingfaceConfig from './huggingface';
 import { cerebrasProviderAPIConfig } from './cerebras';
 import { InferenceNetProviderConfigs } from './inference-net';
 import SambaNovaConfig from './sambanova';
+import { UpstageProviderConfig } from './upstage';
```

Comment:
Please rename this to UpstageConfig to keep with the convention we've been following above (I can see InferenceNet and cerebras are also named this way, but those have to be changed as well)